### PR TITLE
Change the method of obtaining the classpath

### PR DESCRIPTION
**Pull request by JoeKerouac**
Change the method of obtaining the classpath. The old method is to get the URLClassPath(ucp) field in the URLClassLoader. This class is a java class that is not recommended to be used and has no access in JDK9. Therefore, an equivalent implementation has been changed.

### CODE REVIEW

File: sofa-ark-parent/core/spi/src/test/java/com/alipay/sofa/ark/spi/argument/LaunchCommandTest.java [hunk 1]
```diff
@@ -91,8 +106,9 @@ public void testCommandParser() {
             }
 
         } catch (Exception ex) {
-            Assert.assertNull(ex);
+            exception = ex;
```
[the remark this answers is not in the document]
Comment (JoeKerouac) [reply sofa-ark-parent/core/spi/src/test/java/com/alipay/sofa/ark/spi/argument/LaunchCommandTest.java]: oh , why not just throw or print exception when error occur?  error always occur when this Assert run.

File: sofa-ark-parent/core/spi/src/main/java/com/alipay/sofa/ark/spi/tools/RuntimeUtil.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import java.net.URLClassLoader;
+
+/**
+ * runtime utils
+ *
+ * @author joe
+ * @since 0.3.0
+ */
+public class RuntimeUtil {
+    /**
```
[the remark this answers is not in the document]
Comment (JoeKerouac) [reply sofa-ark-parent/core/spi/src/main/java/com/alipay/sofa/ark/spi/tools/RuntimeUtil.java]: ok

File: sofa-ark-parent/core/spi/src/test/java/com/alipay/sofa/ark/spi/argument/LaunchCommandTest.java [hunk 3]
```diff
@@ -80,8 +95,8 @@ public void testCommandParser() {
             Assert.assertTrue(launchCommand.getEntryMethodName().equals(method.getName()));
             Assert.assertTrue(launchCommand.getExecutableArkBizJar().equals(fatJarUrl));
             Assert.assertTrue(launchCommand.isTestMode());
-            Assert.assertTrue(launchCommand.getClasspath().length == classpath
-                .split(CommandArgument.CLASSPATH_SPLIT).length);
+            Assert.assertTrue(launchCommand.getClasspath().length == (classpath.isEmpty() ? 0
+                : classpath.split(CommandArgument.CLASSPATH_SPLIT).length));
```
[the remark this answers is not in the document]
Comment (QilongZhang) [reply sofa-ark-parent/core/spi/src/test/java/com/alipay/sofa/ark/spi/argument/LaunchCommandTest.java]: in windows environment?

[the remark this answers is not in the document]
Comment (JoeKerouac) [reply sofa-ark-parent/core/spi/src/test/java/com/alipay/sofa/ark/spi/argument/LaunchCommandTest.java]: this is my fault = =||

**Comment by QilongZhang** [comment]
@JoeKerouac  hi, it seems that test coverage declines, can you fix it ?

**Comment by JoeKerouac** [comment]
@QilongZhang the coverage declines cause by i merge some code from master , this code is not belong to me.

**Comment by QilongZhang** [comment]
@JoeKerouac 
image

The master is ok..

**Comment by JoeKerouac** [comment]
@QilongZhang I think the coverage rate is the same as that of the previous PR #43 , but it still shows that the coverage rate is decreasing. Can I help you to see why?

**Pull request merged**